Packs/CyberArkEPM/Integrations/CyberArkEPMEventCollector/CyberArkEPMEventCollector.py

```diff
@@ -0,0 +1,470 @@
+            results = client.get_policy_audits(set_id, from_date, limit, next_cursor)
+            policy_audits[set_id]['events'].extend(results.get('events', []))  # type: ignore
+
+        add_fields_to_events(policy_audits[set_id]['events'], 'arrivalTime', 'policy_audits')  # type: ignore
+        policy_audits[set_id]['next_cursor'] = next_cursor or 'start'
+
+    return policy_audits
+
+
+def get_detailed_events(client: Client, last_run_per_id: dict, limit: int) -> dict[str, dict[str, str | list]]:
+    """
+    Args:
+        client (Client): CyberArkEPM client to use.
+        last_run_per_id (dict): A dict of set_ids and a dict of dates and next_cursor from where to get the events.
+            {'123': {'from_date': '01-02-2023T23:20:50Z', 'next_cursor': '123465'}}.
+        limit (int): The maximum events to get.
+    Returns:
+        (dict) A dict of {'set_id': {'events' [list events associated with a list of set names], 'next_cursor': '123456'}}.
+    """
+    detailed_events: dict[str, dict[str, str | list]] = {}
+
+    for set_id, last_run in last_run_per_id.items():
+        detailed_events[set_id] = {'events': [], 'next_cursor': ''}
+        from_date = last_run.get('detailed_events').get('from_date')
+        next_cursor = last_run.get('detailed_events').get('next_cursor')
+
+        results = client.get_events(set_id, from_date, limit, next_cursor)
+        detailed_events[set_id]['events'] = results.get('events', [])
+
+        while (next_cursor := results.get('nextCursor')) and len(detailed_events[set_id]['events']) < limit:
+            results = client.get_events(set_id, from_date, limit, next_cursor)
+            detailed_events[set_id]['events'].extend(results.get('events', []))  # type: ignore
+
+        add_fields_to_events(detailed_events[set_id]['events'], 'arrivalTime', 'detailed_events')  # type: ignore
+        detailed_events[set_id]['next_cursor'] = next_cursor or 'start'
+
+    return detailed_events
+
+
+""" COMMAND FUNCTIONS """
+
+
+def get_admin_audits_command(client: Client, last_run: dict, limit: int | None = 5) -> tuple[list, CommandResults]:
+    """ Fetches admin audits from CyberArkEPM
+    Args:
+        client (Client): CyberArkEPM client to use.
+        last_run (dict): The last run {set_id: {from_date: 01-01-2023T00:00:00.123Z}}.
+        limit (int): The maximum admin audits to return per set_id default is 5.
+    Return:
+        (list, CommandResults) A list of admin audits to push to XSIAM, A CommandResults object.
+    """
+    results = get_admin_audits(client, last_run, limit)  # type: ignore
+    events_list = list(chain(*results.values()))
+    human_readable = tableToMarkdown('Admin Audits', events_list)
+
+    return events_list, CommandResults(readable_output=human_readable, raw_response=events_list)
+
+
+def get_policy_audits_command(client: Client, last_run: dict, limit: int | None) -> tuple[list, CommandResults]:
```

Comment:
get_policy_audits_command and get_detailed_events_command are practically the same no?
consider passing the different method call as an argument and remove on of the methods.
get_policy_audits and get_detailed_events